### PR TITLE
Ol cleanup

### PR DESCRIPTION
**Pull request by austinjpaul**
This will remove the orange line shutdown-related emoji, and change the line graph description for the backend changes.

### CODE REVIEW

**Review by devinmatte — COMMENTED** [review]
~So I feel like we should remove the gridlines, but still keep the indicators on the lines like we kept the derailments. Otherwise that 30 day drop and spike looks odd to a new viewer~

Ignore me, that's exactly what you did

**Review by PatrickCleary — APPROVED** [review]

File: src/slowzones/formattingUtils.ts [hunk 1]
```diff
@@ -11,9 +11,7 @@ const textSize = isMobile ? '11' : '14';
 const DAY_MS = 1000 * 60 * 60 * 24;
 
 export const EMOJI = {
-  'derailment': ` 🚨`,
-  'construction': ` 🚧`,
-  'shutdown': ` ⚠️`,
+  'derailment': ` 🚨`
```
Comment (mathcolo) [inline src/slowzones/formattingUtils.ts:14]: does this need to be a template literal?

**Review by mathcolo — APPROVED** [review]

File: src/slowzones/formattingUtils.ts (hunk 1 above)
Comment (austinjpaul) [reply src/slowzones/formattingUtils.ts:14]: I do think we will come back and add more here later. (MBTA prove me wrong...)

Comment (mathcolo) [reply src/slowzones/formattingUtils.ts:14]: nooo I mean backticks `` vs regular string ''

Comment (austinjpaul) [reply src/slowzones/formattingUtils.ts:14]: Ohhhhh haha sorry. Probably not, that's my bad

**Pull request merged**